protocol/wavefront/wavefrontlistener.go

```diff
@@ -0,0 +1,255 @@
+package wavefront
+
+import (
+	"bufio"
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/datapoint/dpsink"
+	"github.com/signalfx/golib/errors"
+	"github.com/signalfx/golib/log"
+	"github.com/signalfx/golib/pointer"
+	"github.com/signalfx/golib/sfxclient"
+	"github.com/signalfx/metricproxy/logkey"
+	"github.com/signalfx/metricproxy/protocol"
+	"github.com/signalfx/metricproxy/protocol/collectd"
```

Comment:
Not found of this cross-protocol dependency, especially if it's just for the `GetDimensionsFromName`. Could this be extracted?

Reply:
I respectfully disagree. This is our collectd syntax that i'm borrowing for this purpose.